src/perception/aquisition/zed_launch/cone_detection/cone_detection.cpp

```diff
@@ -15,8 +15,6 @@
 #include <NvInfer.h>
 
 using namespace nvinfer1;
-#define NMS_THRESH 0.4
-#define CONF_THRESH 0.8
```

Comment:
why did you remove the cone detection threshold? and what is NMS_THRESH

Reply:
Its in zed_launch.hpp but you're right makes sense for it to be in cone_detection.cpp

Reply:
nms threshold is supposedly how much overlap two detections can have on each other. However, its not being used atm